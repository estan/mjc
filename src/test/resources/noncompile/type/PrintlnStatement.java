class PrintlnStatementTest {
    public static void main(String[] args) {
        int i;
        int[] ia;
        boolean b;
        A cA;
        B cB;

        //System.out.println(i); // OK!
        System.out.println(ia); // INVALID_PRINTLN_TYPE
        System.out.println(b); // INVALID_PRINTLN_TYPE
        System.out.println(cA); // INVALID_PRINTLN_TYPE
        System.out.println(cB); // INVALID_PRINTLN_TYPE
    }
}
class A {}
class B {}
