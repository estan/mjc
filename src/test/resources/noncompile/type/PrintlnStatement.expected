INVALID_PRINTLN_TYPE
INVALID_PRINTLN_TYPE
INVALID_PRINTLN_TYPE
